DocumentFormat.OpenXml.Tests/FlatOpcTests.cs

```diff
@@ -145,6 +147,25 @@ public void CanCreateFlatOpcPresentationDocuments8()
             }
         }
 
+        [Fact(Skip = "#380")]
+        public void CanCreateFlatOpcPresentationDocumentsDynamically()
+        {
+            using (MemoryStream stream = new MemoryStream())
+            using (PresentationDocument presentation = PresentationDocument.Create(stream, PresentationDocumentType.Presentation))
+            {
+                    PresentationPart presentationPart = presentation.AddPresentationPart();
```

Comment:
nit: looks like the whitespace is off. Did you format the doc before committing? ctrl+k+d will do that in VS.

Reply:
I didn't. I'll do it.